### DISCUSSION TITLE
Include version\tag\build in footer

### DISCUSSION THREAD

**Discussion by MartynKeigher** [JordanKnott/taskcafe#66] [Ideas]
**Describe the solution you'd like**
Grafana (for reference) does a pretty nice job of including some info on the build\version\tag that is currently running on the logon screen. Something similar on TaskCafe would be a nice little addon, so that we know what version we are using, especially right now while the project is in active dev.

**Additional context**

![GrafanaFooter](https://user-images.githubusercontent.com/3298134/94092238-357f5a00-fde8-11ea-9ef6-b2ccb08dc1c3.jpg)

**Comment by JordanKnott** [comment]
I have marked this task as a good first issue.

The first step would be to add build information during the build step in the `magefile.go`

A good article can be found [here](https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications).

Then next step would be to create a GraphQL query that sends this information. This involves updating the [gqlgen](https://gqlgen.com/) schema files. After updating the files you can use `mage s` to regenerate the schema.resolver.go implementation.

- `internal/graph/schema/_root.gql`
- `internal/graph/schema.resolvers.go`

That last step would be to add the info to the Auth component.

- `frontend/src/Auth/index.tsx`

Please take a look at the [developers guide](https://github.com/JordanKnott/taskcafe/wiki/Developers-Guide) for an idea on how the project is structured.

**Comment by sai-github** [comment]
Hi, this would be my first time contribution to open source project .. can I take this ?

**Comment by JordanKnott** [comment]
@sai-github Sure

**Comment by sai-github** [comment]
@JordanKnott can I include the footer component under `frontend/src/shared/components/footer` ?

**Comment by JordanKnott** [comment]
@sai-github I would probably place the component in `frontend/src/Auth/Footer.tsx` since it is only really should be used in the login module (at least for now).
